packages/connect-playwright/src/create-mock-router.ts

```diff
@@ -184,18 +191,18 @@ async function universalHandlerToRouteResponse({
 }) {
   const headers = await request.allHeaders();
   const abortSignal = new AbortController().signal;
-  // eslint-disable-next-line @typescript-eslint/no-explicit-any -- The Serializable type isn't exposed by Playwright
-  let body: any;
+  let body: UniversalServerRequest["body"] = [];
```

Comment:
Not sure of the best default value for this.

Reply:
`[]` is not a type error because the body can be pre-parsed JSON, and `[]` is a valid JSON value. 

You are providing the request body `[]` to the call, if the original request had an empty body. That's no good. To provide an empty request body, create an empty byte stream with `createAsyncIterable<Uint8Array>([])`.

Let's also add a test for Connect GET, because I'm pretty sure we would crash on it.